Controllers/HomeController.cs

```diff
@@ -43,7 +43,7 @@ public async Task<IActionResult> Contact()
             var scopes = new string[] { "user.read" };
             try
             {
-                var accessToken = await _tokenAcquisition.GetAccessTokenOnBehalfOfUser(HttpContext, User, scopes);
```

Comment:
The HttpContext already contains a User (ClaimsPrincipal object), so it's not needed.